Lib/fontTools/cffLib/__init__.py

```diff
@@ -2231,6 +2237,20 @@ def decompileAllCharStrings(self, progress):
 				progress.increment(0)  # update
 			i = i + 1
 
+	def recalcFontBBox(self):
+		initialBounds = (float('inf'), float('inf'), float('-inf'), float('-inf'))
+		bounds = initialBounds
```

Comment:
I feel like this could be None initially then below you would do

if bounds is not None:
    unionRect(..)
else:
    bounds = csBounds